blenderproc/python/types/LightUtility.py

```diff
@@ -77,7 +77,7 @@ def set_type(self, light_type: str, frame: Optional[int] = None):
         Utility.insert_keyframe(self.blender_obj.data, "type", frame)
 
     def setup_as_projector(self, pattern: np.ndarray, frame: Optional[int] = None):
-        """ Sets a spotlight source as projector of a pattern image. Sets location and angle of projector to current
+        r""" Sets a spotlight source as projector of a pattern image. Sets location and angle of projector to current
```

Comment:
```suggestion
        """ Sets a spotlight source as projector of a pattern image. Sets location and angle of projector to current
```

Reply:
This is `r"""` is necessary as the string contains `\c` which else would be weirdly interpreted.

Reply:
Ah, makes sense